packages/main/src/components/protected/topbar/styles/TopBarSearchModal.module.css

```diff
@@ -46,7 +46,8 @@
   display: flex;
   align-items: center;
   border-bottom: 0.5px solid lightgrey;
-  padding-bottom: 10px;
+  /* padding-bottom: 10px; */
```

Comment:
If this code isn't being used, do away with it entirely instead of commenting it out

Reply:
Alright, I will do that. Is every other thing alright?

Reply:
> If this code isn't being used, do away with it entirely instead of commenting it out

Done.

Reply:
Hi @Luqman07, everything else is okay. There's other css that you commented out in the `TopBarSearchModal.module.css`. You can do away with it then ask for a review again

Reply:
Hello @JulianaSau, I have made the changes, kindly review it. Thanks.